app/code/Magento/Checkout/view/frontend/web/js/view/cart-item-renderer.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'uiComponent',
+    'escaper'
+], function (Component, escaper) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            allowedTags: ['div', 'span', 'b', 'strong', 'i', 'em', 'u', 'a']
+        },
+
+        /** @inheritdoc */
+        initialize: function () {
+            this._super();
+        },
+
+        /**
+         * Prepare the product name value to be rendered as HTML
+         *
+         * @param {String} productName
+         * @return {String}
+         */
+        getProductNameUnsanitizedHtml: function (productName) {
+            return escaper.escapeHtml(productName, this.allowedTags);
```

Comment:
Looks like now we have 2 times escaping, am I correct? Does it works fine now?

Reply:
just seperate function name for clarify imo. Both function do the same task for escape html string

Reply:
I mean right now product name escaped on backend and on front end , isn’t it?

Reply:
should we return product name ? Current in this PR we have 2 time escape in product name

Reply:
I think it would be good, also please add comment that it’s escaped on backend

Reply:
Also would it complain if we’ll use name `getProductNameHtml`? I think it shouldn’t and it is better name for the function

Reply:
name method need have suffix UnsanitizedHtml otherwise static will raise error
https://github.com/magento/magento2/blob/2.4-develop/dev/tests/static/framework/Magento/Sniffs/Html/HtmlBindingSniff.php#L79

Reply:
Ok, so I think we can leave there something like this:
```suggestion
            // product name already escaped on backend
            return productName;
```